tests/test_cli.py

```diff
@@ -969,10 +744,18 @@ def test_cli_runserver_command_environment_file_generation(monkeypatch):
     def mock_uvicorn_run(_, env_file=None, **kwargs):
         """Mocks uvicorn.run asserting environment file content."""
         with open(env_file, mode="r", encoding=settings.LOCALE_ENCODING) as file:
-            assert file.readlines() == [
+            x = [
```

Comment:
What about the following?

```suggestion
            env_lines = [
```